components/governance/org.wso2.carbon.governance.taxonomy/src/main/java/org/wso2/carbon/governance/taxonomy/services/TaxonomyManager.java

```diff
@@ -129,6 +130,24 @@ JSONArray query(QueryBean taxonomyQueryBean, PaginationBean taxonomyPaginationBe
                 taxonomyPaginationBean.getStartNode(), taxonomyPaginationBean.getEndNode(), nodeList);
     }
 
+    /**
+     * This method will retrieve the results for rest api queries
+     *
+     * @return Json array with processed results
+     * @throws XPathExpressionException
+     * @throws JSONException
+     * @throws TaxonomyException
+     */
+    JSONArray query(QueryBean taxonomyQueryBean)
+            throws XPathExpressionException, JSONException, RegistryException, UserStoreException {
+        JSONArray jsonArray = new JSONArray();
+        JSONObject taxonomyNameObj = new JSONObject();
+        String taxonomyName = queryProvider.getTaxonomyNameById(taxonomyQueryBean);
+        taxonomyNameObj.put("taxonomyName",taxonomyName);
```

Comment:
Thank you Niranjan for mentioning this,
Fixed with this commit https://github.com/wso2/carbon-governance/pull/255/commits/0af3af9362a8cbd2d96daa32d30a16309bd2d352